### ISSUE TITLE
After expanding, text inside the button is truncated in Expander button. 

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#235]
Found during accessibility grading.

Tracking internal bug: https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/40139499
"Users with low vision will not able to consume the content of the button if it is not visible in Text scaling mode."

**Comment by AgneLukoseviciute** [comment]
![image](https://user-images.githubusercontent.com/41223743/175646821-8b4c9456-d435-41a6-8855-78c02aae9209.png)
related: https://github.com/microsoft/react-native-gallery/issues/237
